test/e2e/delete_test.go

```diff
@@ -2,21 +2,90 @@ package e2e
 
 import (
 	"context"
+	"fmt"
+	"strings"
 	"testing"
 
+	config "github.com/openshift/api/config/v1"
 	mapi "github.com/openshift/machine-api-operator/pkg/apis/machine/v1beta1"
+	"github.com/pkg/errors"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
 
+	"github.com/openshift/windows-machine-config-operator/controllers"
 	"github.com/openshift/windows-machine-config-operator/pkg/secrets"
+	win "github.com/openshift/windows-machine-config-operator/pkg/windows"
```

Comment:
Is there a reason for using `win`. I suggest we only introduce aliases when really needed. Applies to all places.

Reply:
FYI, my comment was a question and I would have been satisfied with your answer given this was a situation where the alias was really needed.